File: core-java-practice/gcr-codebase/programming-elements/AddTwoNumbers.java

package Dayone;
import java.util.*;
//Write a program that takes two numbers as input from user and prints their sum
public class AddTwoNumbers {
  public static void main (String [] args){
    Scanner sc=new Scanner(System.in);
    int n =sc.nextInt();
    int m=sc.nextInt();
    int sum=n+m;
    System.out.println("The sum is :" +  sum);
  }
}

File: core-java-practice/gcr-codebase/programming-elements/AddTwoNumbers.cs
using System;

namespace ProgrammingElements{
    // Write a program that takes two numbers as input from user and prints their sum
    class AddTwoNumbers{
        static void Main(string[] args){
            int n = int.Parse(Console.ReadLine());
            int m = int.Parse(Console.ReadLine());
            int sum = n + m;
            Console.WriteLine("The sum is: " + sum);
        }
    }
}
